core/compress_api.php

```diff
@@ -83,6 +83,13 @@ function compress_handler_is_enabled() {
  * @access public
  */
 function compress_start_handler() {
+	# Do not start compress handler if we got any output so far, as this
+	# denotes that an error has occurred. Enabling compression in this case
+	# will likely cause a Content Encoding Error when displaying the page.
+	if( ob_get_contents() ) {
```

Comment:
I didn't try,  but would expect that checking ( ob_get_length() > 0 ) is better in terms of performance.

Do we need this check?
There is one single place where compress_start_handler() is called (core.php) where we do check already for blank output. 
```php
if( ( $t_output = ob_get_contents() ) != '' ) {
	echo 'Possible Whitespace/Error in Configuration File - Aborting. Output so far follows:<br />';
	var_dump( $t_output );
	die;
}
unset( $t_output );

# Start HTML compression handler (if enabled)
require_api( 'compress_api.php' );
compress_start_handler();
```

Of course, there might be output from require_api( 'compress_api.php' );

So, maybe remove the check in core.php and have it just in compress_start_handler is an option.